log_parser/log.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+
+
+class Log:
+    """
+    Class representing a log entry with a message and a timestamp.
```

Comment:
I would drop the "...with a..." part as it delves into details too much in the c'tor and will require us to change when we change the Log class